tests/integration/test_end_to_end_election.py

```diff
@@ -459,17 +459,17 @@ def verify_results(self) -> None:
 
         ciphertext_ballots: List[CiphertextAcceptedBallot] = []
         for ballot in self.ballot_store.all():
-            ballot_name = BALLOT_PREFIX + ballot.object_id
```

Comment:
These instances do not need to be changed. These are unrelated to Party. It just happens to be named ballot_name but it is the name of the file.